client/my-sites/stats/style.scss

```diff
@@ -82,168 +82,6 @@ $font-sf-pro-display: "SF Pro Display", $sans;
 	}
 }
 
-
-.stats-card {
```

Comment:
This is probably fine, but I'd triple check since we still have things like this:

https://github.com/Automattic/wp-calypso/blob/0990db38a4f013587d55908af2796950346d9a2e/packages/components/src/horizontal-bar-list/stats-hero-card.tsx#L7

and:

https://github.com/Automattic/wp-calypso/blob/0990db38a4f013587d55908af2796950346d9a2e/packages/components/src/horizontal-bar-list/stats-card.tsx#L8

Reply:
Oh, this is an oversight, I didn't mean to remove the styles for still-used `.stats-card`. Anyway, in da4fa3bab60 I moved the `min-width` style to a better place, to `stats-card.scss` where all the other `.stats-card` styles live. And inlined the `$stats-card-min-width` variable because it's used only once.